lib/Firtool/Firtool.cpp

```diff
@@ -819,7 +831,7 @@ circt::firtool::FirtoolOptions::FirtoolOptions()
       stripDebugInfo(false), fixupEICGWrapper(false), addCompanionAssume(false),
       disableCSEinClasses(false), selectDefaultInstanceChoice(false),
       symbolicValueLowering(verif::SymbolicValueLowering::ExtModule),
-      disableWireElimination(false) {
+      disableWireElimination(false), lintStaticAsserts(true) {
```

Comment:
missing litXmrsInDesign.

Reply:
Whoops. Fixed in the forcibly updated: https://github.com/llvm/circt/pull/8668/commits/921a4d328a01cb855c6d50573580e8d4465adc3a